packages/blits-create/.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*]
+charset = utf-8
+indent_size = 2
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Do we need this file here? Same goes for the other files on the root level of this package, like .gitignore, but also CHANGELOG, LICENSE, etc

Since it's part of the same project, I would say that it's not needed?